streaming_jupyter_integrations/magics.py

```diff
@@ -46,7 +47,13 @@ def __init__(self, shell: Any):
             "--add-opens=java.base/java.util=ALL-UNNAMED "
             "--add-opens=java.base/java.lang=ALL-UNNAMED"
         )
+        global_conf = self.__read_global_config()
         conf = Configuration()
+        for key, value in global_conf.items():
+            if type(value) is str:
```

Comment:
According to the [PyFlink docs](https://nightlies.apache.org/flink/flink-docs-master/api/python/pyflink.common.html#pyflink.common.Configuration), there are more setters than just string and integer ones. I think it would be better to:
1. add other setters (preferably using `if/elif` or a dictionary of lambdas, e.g.);
2. log a warning or even throw an Exception if a value from the dictionary has no setter available (i.e., type of the value is not recognized by the code)